File: stack01.py
"""
stack - 괄호 맞추기
"""
list = ['(', '(', ')', ')']
stack = []
for i in list:
    if i == '(':
        stack.append('(')
    elif i == ')':
        stack.pop()
    else:
        print('Error!')
if len(stack) > 0:
    print('False!!')
else:
    print('True!!')

File: stack01.js
/* stack -괄호 맞추기*/
const list = ['(', '(', ')']
let stack = [];

for (let i = 0; i < list.length; i++) {
    if (list[i] == '(') {
        stack.push('(');
    } else if (list[i] == ')') {
        stack.pop();
    } else {
        console.log('ERROR!!')
    }
}

if (stack.length > 0) {
    console.log('False!!')
} else {
    console.log('True!!')
}